src/aruku/walking.cpp

```diff
@@ -26,10 +26,14 @@
 #include "math/vector.h"
 
 #include <map>
+#include <iostream>
 #include <string>
 #include <vector>
 #include <cmath>
 #include <cstdlib>
+#include <memory>
+#include <fstream>
+#include <nlohmann/json.hpp>
```

Comment:
Put headers in alphabetical order, external header first (include ROS 2 headers), and then C/C++ STL headers.

e.g.
```c++
#include <nlohmann/json.hpp>

#include <fstream>
#include <memory>
...
```

Reply:
actually hasn't been resolved, but it's okay.